packages/ng-schematics/src/component/index.ts

```diff
@@ -67,7 +67,11 @@ export function singleComponent(templateOptions: TemplateOptions, skipRoute: boo
 }
 export function component(options: ComponentOptions): Rule {
 	return async (tree: Tree, context: SchematicContext) => {
-		App.initialize();
+		App.initialize("angular-cli");
+		GoogleAnalytics.post({
```

Comment:
There is another post in the `ig add` logic that will be skipped here: 
When the user chooses a template, there's a post that [contains information about the choice](https://github.com/IgniteUI/igniteui-cli/blob/vNext/packages/cli/lib/commands/add.ts#L93). This should also be added to the component schematic.